### ISSUE TITLE
[FR]: Retrieval date for articles/items

### ISSUE THREAD

**Issue by birdie-github** [martinrotter/rssguard#947]
### Brief description of the feature request

Would be nice if articles had a field "Retrieval date" when the item was actually downloaded/received.

**Comment by Rex000** [comment]
Yes, a column that could display the exact retrieval date and time could be useful.
